core/src/main/java/org/mapfish/print/servlet/job/impl/ThreadPoolJobManager.java

```diff
@@ -628,8 +635,12 @@ public void run() {
         }
       } catch (javax.persistence.PessimisticLockException e) {
         // Ignore error on pessimistic locking
+        unhealthyCountersHealthCheck.recordUnhealthyProblem(
+            getClass().getSimpleName(), "ignorePessimisticLockException");
```

Comment:
pessimisticLockError

Reply:
As Error has a different connotation in the try/catch world of java, how about pessimisticLockProblem or pessimisticLockIssue ?
I suggest we use a word like Issue or Problem (as nothing is done in the code to address the exception.
And a word like Failure when something is done after identifying something went wrong.